### PR TITLE
issue 35: fixed headings

### LINKED ISSUE #35: Headings
image
image

### PR DESCRIPTION
**Pull request by shrotia**
Fix #35

Test URLs:
- Before: https://main--infosys--aemsites.hlx.live/
- After: https://issue-35-v1--infosys--aemsites.hlx.live/drafts/shroti/iki

### CODE REVIEW

**Comment by shrotia** [comment]
Note: Update of the Collection Library is pending

File: styles/styles.css [hunk 1]
```diff
@@ -22,6 +22,7 @@
   --black-olive: #3f3f3f;
   --bright-blue: #007cc3;
   --dark-denim: #00578A;
+  --chinese-white: #dfe1e2;
```
Comment (anuj-adobe) [inline styles/styles.css:25]: Should we refrain from using color name with a specific ethnicity just to avoid effecting someone.

**Review by anuj-adobe — APPROVED** [review]
LGTM

**Comment by anuj-adobe** [comment]
psi check is failing due to low performance score.

**Comment by shrotia** [comment]
> * https://main--infosys--aemsites.hlx.live/

@anuj-adobe seems Hero-Slider component is causing this low score for Mobile. Independent of this PR. Let's tract & fix it separately.

https://pagespeed.web.dev/analysis/https-main--infosys--aemsites-hlx-live-drafts-shroti-hero-slider/o543hed3tq?form_factor=mobile

https://main--infosys--aemsites.hlx.live/drafts/shroti/hero-slider

Logged https://github.com/aemsites/infosys/issues/38

**Comment by shrotia** [comment]
> psi check is failing due to low performance score.

@anuj-adobe seems Hero-Slider component is causing this low score for Mobile. Independent of this PR. Let's tract & fix it separately.

https://pagespeed.web.dev/analysis/https-main--infosys--aemsites-hlx-live-drafts-shroti-hero-slider/o543hed3tq?form_factor=mobile

https://main--infosys--aemsites.hlx.live/drafts/shroti/hero-slider

Logged https://github.com/aemsites/infosys/issues/38

**Comment by shrotia** [comment]
Added sections in the Block Library: https://main--infosys--aemsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/tools/sidekick/blocks/sections&index=0

**Pull request merged**